.github/workflows/_check-build-liminal-node.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This workflow checks build of aleph-node feature liminal enabled
+name: Build liminal node
+on:
+  workflow_call:
+
+jobs:
+  main:
+    name: Build liminal node
+    runs-on: self-hosted
+    steps:
+      - name: Checkout aleph-node source code
+        uses: actions/checkout@v3
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+
+      - name: Install WASM target
+        run: rustup target add wasm32-unknown-unknown
+
+      - name: Build liminal binary and runtime
+        run: |
+          cargo check --release -p aleph-node --features "liminal"
```

Comment:
I know that I had told you that `cargo check` would suffice but there are some tests that are liminal-specific so we should run them _somewhere_ as well. /cc @pmikolajczyk41

Reply:
No problem, should I change it to `cargo build` ? Actually even better, I can re-use the same workflow here

Reply:
No, not `cargo build`. I was thinking about tests. Let's wait for @pmikolajczyk41 as he's more familiar with what we have where.

Reply:
we could run just runtime tests, like: https://github.com/Cardinal-Cryptography/aleph-node/pull/872/files (so that we don't waste time for tests that are already run in other places)

Reply:
That will significantly affect the shape of this PR. Let's discuss this offline.

Reply:
@pmikolajczyk41 can you ticket it please?

Reply:
https://github.com/Cardinal-Cryptography/aleph-node/pull/1078 this should be enough